File: FinishedProject/CF Finder/Views/ATMDetailView/OtherCFTableViewCell.m
//
//  Coffeeshop Finder
//
//  Created by Duc Nguyen on 9/5/16.
//  Copyright © 2016 Duc Nguyen. All rights reserved.
//

#import "OtherCFTableViewCell.h"
#import "Constants.h"

@implementation OtherCFTableViewCell

-(void)layoutSubviews
{
    [super layoutSubviews];

    // Fonts
    [self.nameLabel setFont:[UIFont fontWithName:FONT_HELVETICA_MEDIUM size:16]];
    [self.addressLabel setFont:[UIFont fontWithName:FONT_HELVETICA_REGULAR size:16]];
    
    // Text Color
    [self.addressLabel setTextColor:UIColorFromRGB(RGB_GRAY)];
    
    // Corner Radius
    [self.categoryIconImageView.layer setCornerRadius:5];
    [self.categoryIconImageView setClipsToBounds:YES];
    [self.categoryIconImageView setBackgroundColor:UIColorFromRGB(RGB_GRAY)];
}


@end

File: FinishedProject/CF Finder/Views/OtherCFTableViewCell.swift
//
//  OtherCFTableViewCell.swift
//  CF Finder
//
//  Created by Duc Nguyen on 9/7/16.
//  Copyright © 2016 Duc Nguyen. All rights reserved.
//

import Foundation
import UIKit

class OtherCFTableViewCell: UITableViewCell {
    @IBOutlet weak var categoryIconImageView: UIImageView!
    @IBOutlet weak var nameLabel: UILabel!
    @IBOutlet weak var addressLabel: UILabel!
    
    override func layoutSubviews() {
        super.layoutSubviews()
        
        // 
        let constant = Constants.sharedInstance
        
        // Fonts
        self.nameLabel.font = UIFont(name: constant.FONT_HELVETICA_MEDIUM, size: 16)
        self.addressLabel.font = UIFont(name: constant.FONT_HELVETICA_REGULAR, size: 16)
        // Text Color
        self.addressLabel.textColor = constant.UIColorFromRGB(constant.RGB_GRAY)
        // Corner Radius
        self.categoryIconImageView.layer.cornerRadius = 5
        self.categoryIconImageView.clipsToBounds = true
        self.categoryIconImageView.backgroundColor = constant.UIColorFromRGB(constant.RGB_GRAY)
    }
}